Arriba/Arriba.Web/jsx/DetailsField.jsx

```diff
@@ -1,21 +1,25 @@
 // Writes out a field with label and highlighting, but only if the value is set
 export default React.createClass({
+    urlReplace: function (urlFormat, marker, id) {
+        var idForUrl = encodeURIComponent(id);
+        return urlFormat.replace(new RegExp(marker), idForUrl);
+    },
     render: function () {
         if (!this.props.value || this.props.value === "N/A" || this.props.value === "Unknown") return null;
-
-        var link = null;
-        if(this.props.url) link = <DetailReportLink url={this.props.url} id={this.props.id} title={this.props["url-title"]} />;
-
+        
         var content = <span dangerouslySetInnerHTML={highlight(this.props.value)} title={this.props.columnName + ':' + stripHighlight(this.props.value)} />;
         if (this.props.bold) content = <b>{content}</b>;
 
+        var link = null;
+        if(this.props.url) content = <a target="_blank" href={this.urlReplace(this.urlReplace(this.props.url, "{ID}", this.props.id), "{Value}", this.props.value)} title={this.props.title}>{content}</a>;
+
```

Comment:
`this.replace(/{ID}/g, encodeURIComponent(this.props.id)).replace(/{Value}/g, encodeURIComponent(this.props.value))`